packages/form-js-editor/src/features/properties-panel/entries/TextEntry.js

```diff
@@ -2,32 +2,47 @@ import { get } from 'min-dash';
 
 import { useService, useVariables } from '../hooks';
 
-import { FeelTextAreaEntry, isFeelEntryEdited } from '@bpmn-io/properties-panel';
+import { FeelTemplatingEntry, isFeelTemplatingEntryEdited } from '@bpmn-io/properties-panel';
+import { simpleBoolEntryFactory } from './factories';
 
 
 export default function TextEntry(props) {
   const {
     editField,
+    getService,
     field
   } = props;
 
   const {
     type
   } = field;
 
+  const templating = getService('templating');
+
   if (type !== 'text') {
     return [];
   }
 
-  return [
+  const entries = [
     {
       id: 'text',
       component: Text,
       editField: editField,
       field: field,
-      isEdited: isFeelEntryEdited
+      isEdited: isFeelTemplatingEntryEdited
     }
   ];
+
+  if (templating.isTemplate(field.text)) {
+    entries.push(simpleBoolEntryFactory({
```

Comment:
I think a description would be helpful here, as I think it's hard to gather what "strict templating" means.

Reply:
As discussed, we'll wait for feedback on this one as it is kind of hard to give descriptions for, most likely we'll cover this in the documentation and maybe tooltips.